### DISCUSSION TITLE
I need some help with a touch screen

### DISCUSSION THREAD

**Discussion by tuzbtla** [labwc/labwc#2390] [Q&A]
On the touch screen, text was selected when I swiped directly, but I had to scroll through my window, like a mouse wheel.  So I wondered if I could scroll through the window with a two-finger swipe, but as far as I know, libinput doesn't explain touch screen gestures, and it seems that labwc doesn't support touch screen gestures, which seems to require me to implement myself in my application.  I don't know much about this area, and I would like to get some advice on whether my idea is correct or if there are other ways to achieve it.What should I study?

**Comment by johanmalm** [comment]
I'd start with

```
<touch mouseEmulation="no|yes"/>
```

https://labwc.github.io/labwc-config.5.html#touch
